src/project/manifest.rs

```diff
@@ -1439,6 +1479,27 @@ mod test {
         assert_debug_snapshot!(manifest.parsed.target);
     }
 
+    #[test]
+    fn test_remove_pypi_dependencies() {
+        let pixi_cfg = r#"[project]
+name = "pixi_fun"
+version = "0.1.0"
+channels = []
+platforms = ["linux-64", "win-64"]
+
+[dependencies]
+python = ">=3.12.1,<3.13"
+
+[pypi-dependencies]
+jax = { version = "*", extras = ["cpu"] }
```

Comment:
I tested it locally with `requests = "*"` which failed. So maybe extend the test with:

```
requests = "*"
xpackage = "==1.2.3"
ypackage = {version = ">=1.2.3"}
```

Reply:
done